cpmonitor/views.py

```diff
@@ -455,6 +456,14 @@ def markdown_uploader_view(request):
         settings.MARTOR_UPLOAD_PATH, time.strftime("%Y/%m/%d/"), img_uuid
     )
     def_path = default_storage.save(tmp_file, ContentFile(image.read()))
+
```

Comment:
This is perfectly fine, since optimizations are not to be done to early.
I just wonder: Would it be possible to do a single save here? E.g. let `thumbnail()` work on the file in memory?

Reply:
Possibly yes, see https://stackoverflow.com/questions/3723220/how-do-you-convert-a-pil-image-to-a-django-file
However,  the guy there says it took him the better part of a day to find the solution, so I'd rather not (even though the solution is given, it might still require attention to fiddly details).